test/UnitTest/Microsoft.Test.AspNet.OData/Formatter/ODataMediaTypeFormattersTests.cs

```diff
@@ -673,6 +673,50 @@ public void TestCreate_DollarFormat_Error(string dollarFormatValue, string expec
             Assert.Equal(MediaTypeHeaderValue.Parse(expectedMediaType), mediaType);
         }
 
+#if NETCORE
+        [Fact]
+        public void TestGet_InputFormatterMetadata_DontError()
+        {
+            // Arrange
+            var formatters = CreateInputFormatters().ToList();
+            Assert.NotNull(formatters); // Guard assertion
+
+            // Act
+            foreach (var formatter in formatters)
+            {
+                formatter.SupportedMediaTypes.Clear();
+            }
+
+            // Assert
+            foreach (var formatter in formatters)
+            {
+                var exception = Record.Exception(() => formatter.GetSupportedContentTypes("application/json", typeof(string)));
+                Assert.Null(exception);
```

Comment:
Add another `Assert` for the expected value when calling `GetSupportedContentTypes`